spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1083,113 +1083,70 @@ being set to `true`.
-a discovery page for available endpoints.  For example, there is a `health` endpoint
-that shows application health information and an `env` that lists properties from
-Spring's `ConfigurableEnvironment`.  By default, only the health and application info
-endpoints are accessible.  The other endpoints are considered to be sensitive
-and need to be http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-customizing-endpoints[enabled explicitly via configuration].  If you enable
-sensitive endpoints, you should also
-<<configuration-security,secure the Data Flow server's endpoints>> so that
-information is not inadvertently exposed to unauthenticated users.  The local Data Flow server has security disabled by default, so all actuator endpoints are available.
+The Actuator library adds HTTP endpoints under the context path `/management` which is a discovery page for available managerment endpoints.
+For example, there is a `health` endpoint that shows application health information and an `env` that lists properties from Spring's `ConfigurableEnvironment`.
+By default, only the health and application info endpoints are accessible.  The other endpoints are considered to be sensitive
+and need to be  https://docs.spring.io/spring-boot/docs/1.5.12.RELEASE/reference/htmlsingle/#production-ready-customizing-endpoints[enabled explicitly via configuration].
+If you enable sensitive endpoints, you should also <<configuration-security,secure the Data Flow server's endpoints>> so that information is not inadvertently exposed to unauthenticated users.
+The local Data Flow server has security disabled by default, so all actuator endpoints are available.
 
-The Data Flow server requires a relational database, and, if the feature toggled for
-analytics is enabled, a Redis server is also required.  The Data Flow server
-autoconfigures the https://github.com/spring-projects/spring-boot/blob/v1.4.1.RELEASE/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/DataSourceHealthIndicator.java[DataSourceHealthIndicator] and https://github.com/spring-projects/spring-boot/blob/v1.4.1.RELEASE/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/RedisHealthIndicator.java[RedisHealthIndicator] if needed.  The health of these two services is incorporated to the overall health status of the server through the `health` endpoint.
+The Data Flow server requires a relational database, and, if the feature toggled for analytics is enabled, a Redis server is also required.
+The Data Flow server autoconfigures the https://github.com/spring-projects/spring-boot/blob/v1.4.1.RELEASE/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/DataSourceHealthIndicator.java[DataSourceHealthIndicator] and https://github.com/spring-projects/spring-boot/blob/v1.4.1.RELEASE/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/health/RedisHealthIndicator.java[RedisHealthIndicator] if needed.
```

Comment:
The Boot link point to `v1.4.1.RELEASE` - might be dated content.

Reply:
added to https://github.com/spring-cloud/spring-cloud-dataflow/issues/2148